spec/models/user_spec.rb

```diff
@@ -10,20 +23,15 @@
   it 'invalido sem email' do
     user = User.new
     user.username = "teste"
-    user.password_digest = "senha"
+    user.password = "senha"
```

Comment:
Nesse trecho que eu acho que eu "desfiz" a questão de segurança.

- Aqui pode ser só a questão de o teste estar enviando uma string para o controller e não ter nada relacionado à segurança, e não se testa a questão do hash.

Reply:
Acho que está correto, analisando o código parece que você adiciona esse campo e automaticamente é convertido em password_digest.

Reply:
Quanto à testar o hash, me parece que isso é feito nos testes de login e não cadastro de usuário

https://stackoverflow.com/questions/16603559/rspec-the-passwords-in-my-test-are-not-matching-up